lib/common/timerwheel.c

```diff
@@ -159,10 +159,6 @@ int h2o_timerwheel_validate(h2o_timerwheel_t *ctx)
 
 uint64_t h2o_timerwheel_get_wake_at(h2o_timerwheel_t *ctx)
 {
-#if H2O_TIMER_VALIDATE
-    assert(h2o_timer_validate_wheel(w));
-#endif
-
```

Comment:
I think we might want to keep these.

Generally speaking, we would want to keep certain `#if MACRO` even when we do not provide a special interface for setting that MACRO. When debugging, it is always helpful to set certain macro names so that these extra checks would be performed.

Reply:
Thanks for the review.

the assert is removed because  compiler error when manually enable `H2O_TIMER_VALIDATE`.
`
lib/common/timerwheel.c:163:37: error: ‘w’ undeclared (first use in this function)
     assert(h2o_timer_validate_wheel(w));
`

Are you really sure want to keep this?

Reply:
Oh. You are absolutely correct. I would appreciate it if you could change these assertions to `assert(h2o_timerwheel_validate(ctx));`.